app/src/main/AndroidManifest.xml

```diff
@@ -17,14 +17,12 @@
     </queries>
 
     <application
-        android:name="org.tiqr.TiqrApplication"
+        android:name="org.nl.SampleApplication"
```

Comment:
We should decide if we really need it in this project. If we use our own applications in tiqr and eduID, and don't extend it, maybe remove it?

Reply:
also org.nl is wrong, it would be org.sample

Reply:
@pmeulen thought it's a good idea to keep it as a playground app, but i can remove it also no problem

Reply:
@sarahachem ah, yeah, you're right, keep it